PhoneBook.Doc415/DataAccess.cs

```diff
@@ -0,0 +1,52 @@
+﻿using PhoneBook.Doc415.Context;
+using PhoneBook.Doc415.Models;
+
+namespace PhoneBook.Doc415;
+
+internal class DataAccess
+{
+    public void AddContact(string _name, string _email, string _phone, string _title)
+    {
+        Contact newContact= new Contact();
+        newContact.Name= _name;
+        newContact.Email= _email;
+        newContact.Title= _title;
+        newContact.PhoneNumber= _phone;
+
+        try
+        {
+            var db = new PhoneBookContext();
+            db.Contacts.Add(newContact);
+            db.SaveChanges();
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine($"There was an error saving contact: {ex.Message}");
+        }
+    }
+
+    public List<Contact> GetContacts()
```

Comment:
you can use `async` versions for communication with DB

Reply:
Yes,  and why?

Reply:
https://stackoverflow.com/questions/54825879/why-would-i-use-async-instead-of-non-async-method